tests/mock_crust/churn.rs

```diff
@@ -181,20 +219,26 @@ fn request_during_churn_node_to_group() {
         let data_id = data.identifier();
         let message_id = MessageId::new();
 
+        let except_index = added_index.unwrap_or(nodes.len());
+        let quorum_before = quorum(nodes.iter()
+            .enumerate()
+            .filter(|&(i, n)|
+                i != except_index && n.routing_table().is_recipient(&dst.to_destination()))
```

Comment:
maybe an utility function of `target_group_except`? and then move this part of code into it?
otherwise creating such a group will already be a headache. :(